### ISSUE TITLE
[Bug] Corrupt Icon in Latest MacOS Build

### ISSUE THREAD

**Issue by richdenton** [unknownskl/xbox-xcloud-client#1185]
### Describe the bug

The icon of the MacOS 2.2.1 build incorrectly displays as a pixelated mess of colors.
![Screenshot 2024-04-02 at 2 58 01 PM](https://github.com/unknownskl/greenlight/assets/19931034/0d4bd1e7-ff49-4c13-a9e7-cbc97a4131ef)

### To Reproduce

1. Locate the app in Finder
2. Right click on Greenlight
3. Click "Get Info"
4. Note the icon in the top left corner of the dialog

### Expected behavior

The icon should appear as a green cloud on a black background.

### Environment

```markdown
- OS: MacOS
- Device: Mac
- Greenlight version: 2.2.1
```

### New issue

- [X] I confirm that this is a new issue and not a duplicate

**Comment by unknownskl** [comment]
Have you tried to redownload the file? This bug is not visible on my osx.

**Comment by richdenton** [comment]
Yes. I’ve tried redownloading and clearing Finder’s icons cache. Same result, unfortunately.
